esirK/ShoppingListFrontEndReact
# Develop to Master

# What this PR does
Adds the following Functionalities to master

1. User registration and authentication(Login/Logout)
2. Allow user to perform all **CRUD** operations for the shopping lists
3. Allow user to perform all **CRUD** operations for the shopping list items
4. Add pagination for shopping lists
5. Add search functionality for the shoppinglists
6. Add share shoppinglist functionality
7. Add tests for the app
## How should this be tested manually
1. The fully functioning app can be found [here](https://shoppinglist-react-frontend.herokuapp.com)
2. After cloning the repo as per readme, run `npm test` to see the tests run and see the tests coverage
